sqs/src/main/scala/akka/stream/alpakka/sqs/ControlledThrottling.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.sqs
+
+import java.time.Clock
+
+import akka.stream.Attributes
+import akka.stream.impl.fusing.GraphStages.SimpleLinearGraphStage
+import akka.stream.stage.{GraphStageLogic, InHandler, OutHandler, TimerGraphStageLogic}
+
+import scala.concurrent.duration.FiniteDuration
+
+class ControlledThrottling[T](clock: Clock, delay: FiniteDuration) extends SimpleLinearGraphStage[T] {
```

Comment:
This helper stage should move to the `impl` package and be marked as `InternalApi`.